02.wo_Defects/overrun_st.c

```diff
@@ -19,7 +19,7 @@ void overrun_st_001 ()
 {
 	char buf[5];
 	buf[4] = 1; /*Tool not should detect this line as error*/ /*No ERROR: buffer overrun */
-        sink = buf[idx];
+        sink = buf[4];
```

Comment:
we need to read the actual index that was written to otherwise we are reading an indeterminate value which is undefined.